cmd/umount.go

```diff
@@ -91,5 +96,89 @@ func umount(ctx *cli.Context) error {
 	setup(ctx, 1)
 	mp := ctx.Args().Get(0)
 	force := ctx.Bool("force")
+	if !force {
```

Comment:
This should be enabled using a flag (--wait-flush), or we changed the previous behavior.

Reply:
Done